src/Service/Factory.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Jahuty\Service;
+
+use Jahuty\Client;
+
+/**
+ * Factory for API resource services.
+ *
+ * @property  Snippet  $snippets
+ */
+class Factory
+{
+    private $client;
+
+    private static $classes = [
```

Comment:
I would move this static property above the non-static private ones : - )

Reply:
👍 thanks!